dot-net-sdk/http/EppoHttpClient.cs

```diff
@@ -59,6 +74,11 @@ public void AddDefaultParam(string key, string value)
         };
 
         parameters.ToList().ForEach(x => request.AddParameter(new QueryParameter(x.Key, x.Value)));
+
+        if (lastVersion != null)
+        {
+            headers.Add("IF-NONE-MATCH", lastVersion);
```

Comment:
Nice, so this will have fastly not even return the resource if it hasn't changed? Consider adding a comment about this header

Reply:
done